tests/pytests/test_missing.py

```
@@ -554,7 +554,7 @@ def testMissingGC():
     res = env.cmd('FT.INFO', 'idx')
     gc_sec = res[res.index('gc_stats') + 1]
     bytes_collected = gc_sec[gc_sec.index('bytes_collected') + 1]
-    env.assertTrue(int(bytes_collected) > 0)
+    env.assertGreater(int(bytes_collected), 0)
```

Just annoyed you? 😅

---

I think it failed at some point and just printed the value was false instead of printing the actual value of the bytes...